account-decoder/src/parse_account_data.rs

```diff
@@ -20,34 +20,26 @@ use {
     thiserror::Error,
 };
 
-lazy_static! {
-    static ref ADDRESS_LOOKUP_PROGRAM_ID: Pubkey = address_lookup_table::id();
-    static ref BPF_UPGRADEABLE_LOADER_PROGRAM_ID: Pubkey = bpf_loader_upgradeable::id();
-    static ref CONFIG_PROGRAM_ID: Pubkey = config::id();
-    static ref STAKE_PROGRAM_ID: Pubkey = stake::id();
-    static ref SYSTEM_PROGRAM_ID: Pubkey = system_program::id();
-    static ref SYSVAR_PROGRAM_ID: Pubkey = sysvar::id();
-    static ref VOTE_PROGRAM_ID: Pubkey = vote::id();
-    pub static ref PARSABLE_PROGRAM_IDS: HashMap<Pubkey, ParsableAccount> = {
+pub static PARSABLE_PROGRAM_IDS: std::sync::LazyLock<HashMap<Pubkey, ParsableAccount>> =
```

Comment:
Sorry for the churn, but I saw the opportunity to clean this one up a bit further. Once https://github.com/anza-xyz/agave/pull/6096 lands, you should be able to rebase and the diff for this file should be pretty slim.

I'll comment here when https://github.com/anza-xyz/agave/pull/6096 is merged

Reply:
And 6096 has merged

Reply:
Ha oops, we had two of these. I confused this one with the one in `transaction-status/src/parse_instruction.rs`. You've dealt with enough rebases, so I'll clean it up after this one (similar to #6096)